_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At forward financing how we work is something we are always trying to improve on.
+We have some core tenants to how we work as a team, our director of engineering,
+Patrick Hereford, wrote a great post about that recently. You should check it out [here](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'}
+
+We have been in the middle of building the team’s numbers here,
+in a push that will eventually more than double the size or our group.
+These positions are for different levels of engineers and for both local and remote team members.
+This is a very exciting time for us, but how successful we are really depending on who we are looking for.
+
+What we care about it finding a team member is the interesting in joining a blameless culture that is rooted in learning as we build the product the business needs, which we mainly do with code review and pairing.
+This interest in being part of that is really the paramount concern.
+There are lots of great engineers out there, all with their own styles and preferred ways of working in a team.
+We really wanted to focus on finding team members who would be adding to and strengthening the values we have in place.
+
+After talking about technical challenges, and how we might tailor something to be most useful to what we needed,
+we coalesced on pair programming. We do not practice strict formal pairing here as a whole,
+but we really like to collaborate, especially on large features.
+It is really common for each person to either hop on a call with another engineer about a challenge, an idea, or to talk over code review. Settling on that as the common factor, being able to communicate well with any new person, was really key.
+
+In the end we set up a process for pairing with a candidate that could be done in about an hour, and had both a back-end and front-end section (this was important for us because as a team, we are all full-stack).
+We choose to go with a three person call, one teammate leading the frontend, and another teammate to lead the backend portion. This worked out well for us because then two engineers get to talk to the candidate, and the candidate also gets to ask questions and see how we work with each other. The idea is to give a small window into how we work, while also getting an idea about how the candidate might fit into that with us. Our setup is based on some cloned codebases of ours with a few features removed. This worked for us because it was nice to see a candidate work with code that is based on our real codebase, and also makes it a little simpler for us keep in mind some of ways to fill in the missing code.
```

Comment:
• three-person —

• Line break between: `each other. The idea`

• candidate might fit ~into that with us~

• Our setup is based on our own codebases, with a few features removed

• ~This worked for us because~ It was nice to see a candidate work with code that is based on our ~real~ `actual` codebase, and also `makes` made it ~a little simpler for us keep in mind some of ways to fill in the missing code~ easier for us to provide guidance toward possible solutions.

Reply:
✔️ 🍰